sdk/identity/Azure.Identity/src/DeviceCodeCredential.cs

```diff
@@ -106,7 +106,7 @@ public virtual AuthenticationRecord Authenticate(CancellationToken cancellationT
         /// Interactively authenticates a user via the default browser.
         /// </summary>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>The <see cref="AuthenticationRecord"/> which can be used to silently authenticate the account on future execution of credentials using the same <see cref="DeviceCodeCredentialOptions.TokenCache"/>.</returns>
+        /// <returns>The <see cref="AuthenticationRecord"/> which can be used to silently authenticate the account on future execution of credentials using the same <see cref="DeviceCodeCredentialOptions.TokenCachePersistenceOptions"/>.</returns>
```

Comment:
This sounds a bit awkward with `TokenCachePersistenceOptions`. Perhaps we should reword it.